src/OpenTelemetry.Api/README.md

```diff
@@ -77,32 +90,57 @@ specification](https://github.com/open-telemetry/opentelemetry-specification/blo
 Even though `Activity` enables all the scenarios OpenTelemetry supports, users
 who are already familiar with OpenTelemetry terminology may find it easy to
 operate with that terminology. For instance, `StartSpan` may be preferred over
-`StartActivity`. To help with this transition, the OpenTelemetry.API package
-has [shim](#instrumenting-a-libraryapplication-with-opentelemetryapi-shim)
-classes to wrap around the .NET `Activity` classes.
+`StartActivity`. To help with this transition, the OpenTelemetry.API package has
+[shim](#instrumenting-using-opentelemetryapi-shim) classes
+to wrap around the .NET `Activity` classes.
 
 The shim exist only in the API. OpenTelemetry SDK for .NET will be operating
 entirely with `Activity` only. Irrespective of whether shim classes or
 `Activity` is used for instrumentation, the end result would be same. i.e
 Processors/Exporters see the same data.
 
+The recommended way of instrumenting is by using the [.NET Activity
+API](#instrumenting-a-libraryapplication-with-net-activity-api). Users are
+required to just take dependency on the
+[DiagnosticSource](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/5.0.0).
+Adding dependency to
+[OpenTelemetry.API](https://www.nuget.org/packages/opentelemetry.api) is
+required only for the following scenarios.
+
+1. You want to use terminology matching OpenTelemetry spec (Span vs Activity).
+   The [shim](#instrumenting-using-opentelemetryapi-shim) can be useful for such
+   users. Refer to the [comparison of Activity API and OpenTelemetry Tracing
+   API](https://github.com/open-telemetry/opentelemetry-dotnet/issues/947) if
```

Comment:
Do we have documentation anywhere for the handful of functionality exposed via extension methods on Activity? I'm thinking like SetStatus, RecordException, etc. Not sure it makes sense to include this in this doc or perhaps #947.

Reply:
SetStatus is documented in this doc.
I'll review and add RecordException, once I confirm https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/exceptions.md spec is official and are no plans to change it. (I vaguely recollect open issues about, so will confirm that)